cli/hzn.go

```diff
@@ -533,6 +533,21 @@ Environment Variables:
 	devDependencyRemoveCmd := devDependencyCmd.Command("remove", msgPrinter.Sprintf("Remove a project dependency."))
 
 	agbotCmd := app.Command("agbot", msgPrinter.Sprintf("List and manage Horizon agreement bot resources."))
+
+	agbotCacheCmd := agbotCmd.Command("cache", msgPrinter.Sprintf("Manage cached agbot-serving organizations, patterns, and deployment policies."))
+	agbotCacheServedOrg := agbotCacheCmd.Command("servedorg", msgPrinter.Sprintf("List served pattern orgs and deployment policy orgs."))
+	agbotCacheServedOrgList := agbotCacheServedOrg.Command("list", msgPrinter.Sprintf("Display served pattern orgs and deployment policy orgs."))
+	agbotCachePattern := agbotCacheCmd.Command("pattern", msgPrinter.Sprintf("List patterns cached in the agbot."))
+	agbotCachePatternList := agbotCachePattern.Command("list", msgPrinter.Sprintf("Display served patterns cached in the agbot."))
+	agbotCachePatternListOrg := agbotCachePatternList.Arg("org", msgPrinter.Sprintf("Display patterns under this org.")).String()
```

Comment:
This should probably be done with the -o flag like all the other commands.

Reply:
Done.